src/store/server/updateManager/types.ts

```diff
@@ -1,61 +1,68 @@
-export interface ServerUpdateMangerState {
-    version_info: {
-        // eslint-disable-next-line
-        [key: string]: any
+export interface ServerUpdateManagerState {
+    busy: boolean
+    github_rate_limit: number | null
+    github_requests_remaining: number | null
+    github_limit_reset_time: number | null
+    git_repos: ServerUpdateManagerStateGitRepo[]
+    web_repos: ServerUpdateManagerStateGitRepo[]
+    system: {
+        package_count: number
+        package_list: string[]
     }
     updateResponse: {
         application: string
         complete: boolean
-        messages: ServerUpdateMangerStateMessages[]
+        messages: ServerUpdateManagerStateMessages[]
     }
 }
 
-export interface ServerUpdateMangerStateMessages {
+export interface ServerUpdateManagerStateMessages {
     date: Date
     message: string
 }
 
-export interface ServerUpdateMangerStateVersionInfoGitRepoCommits {
+export interface ServerUpdateManagerStateGitRepoCommit {
     sha: string
     author: string
-    date: string
+    date: number
```

Comment:
According the moonraker docs and test, this is a linux epoch timestamp as string. 

```suggestion
    date: string
```